src/components/containers/CloudNativeWay.tsx

```diff
@@ -0,0 +1,108 @@
+// import { Layout, ResponsiveRow } from '../components/layout'
+import React from "react";
+import styled from "styled-components";
+import { ResponsiveRow, ResponsiveRowMD } from '../layout'
+import { Heading, SubHeading, Paragraph } from '../texts/index'
+import { Section } from "../../components/section-layout";
+import { useTheme } from '../../styles'
+
+const Features = styled.div`
+//   min-width: 16rem; big
+  width: 100%;
+  padding: 10px;
+  text-align:start;
+`;
+
+const TwoFeatures = styled.div`
+    min-width: 20rem;
+    // width: 200% !important; big
+    display:flex;
+    flex-flow:wrap row;
+    justify-content:flex-start;
+    align-items:center;
+    align-self:space-around;
+    padding:50px;
+`
+
+
+const CloudNativeWayDiv: React.FC = ({ children }) => {
+    const { sm } = useTheme().screens
+    if (sm) {
+      return <div style={{ margin: '0 1rem' }}>{children}</div>
+    } else {
+      return (
+        <div style={{ margin: '28rem 0 3rem 0', textAlign: 'center' }}>
+          {children}
+        </div>
+      )
+    }
+  };
+
+const CloudNativeWay: React.FC = () => {
+    return (
+        <Section>
+            <CloudNativeWayDiv>
+                <Heading>Do it the cloud-native way</Heading>
+                <>
```

Comment:
a fragment is not required here
read more about fragments here: https://reactjs.org/docs/fragments.html